### DISCUSSION TITLE
my total balance shows 0

### DISCUSSION THREAD

**Discussion by HAN00586** [Chia-Network/chia-blockchain#16176] [Support]
recently I found my Chia wallet Spendable Balance gose to 0 , and there are no any transactions in my history list thesedays. it should be 7 XCH coins in my total balance because I won 8 times through farming and I got 16 XCH, I sent 9 XCH coins to my Exchange account  and left 7 XCH coins in my wallet , but it shows 0 now. what happend Please.

![K_W`357`Y$2R_56@OC{Y40I](https://github.com/Chia-Network/chia-blockchain/assets/143303702/1e80dec0-16e1-4831-bded-e95918c5cfae)

**Comment by shaneo257** [comment]
have you gone to settings > advance and next to Resync Wallet, click on the Resync button, then restart the client

**Reply by HAN00586** [reply]
thanks a lot , it shows the right way now, but it seems that someone has stolen my 7 coins in 16 August in my history list  , any solutions to get my coins back? thank you.

**Reply by shaneo257** [reply]
I am sorry to hear that the coins were stolen and urge you to review our article regarding how to be a hard target:
https://www.chia.net/2021/05/28/securing-your-chia-how-to-be-a-hard-target/ .

The chia blockchain is fully decentralized meaning we at Chia Network do not have any ability to return stolen or lost funds and are unable to assist in the recovery of funds.
